pkg/apis/securityenforcement/client/clientset/versioned/clientset.go

```diff
@@ -53,14 +51,9 @@ func (c *Clientset) Discovery() discovery.DiscoveryInterface {
 }
 
 // NewForConfig creates a new Clientset for the given config.
-// If config's RateLimiter is not set and QPS and Burst are acceptable,
-// NewForConfig will generate a rate-limiter in configShallowCopy.
 func NewForConfig(c *rest.Config) (*Clientset, error) {
 	configShallowCopy := *c
 	if configShallowCopy.RateLimiter == nil && configShallowCopy.QPS > 0 {
-		if configShallowCopy.Burst <= 0 {
```

Comment:
Interested in why this generated code has changed?

Reply:
It looks like it was unnecessary to regenerate the code so merging this change is optional, curious about this change too, my thinking is that I followed the documented process with up to date code generator so likely that future dode gens will do the same. Will look at blame for said lines.

Reply:
looks like these lines were introduced here: https://github.com/IBM/portieris/pull/111, they appear to be connected to rate limiting calls to the apiserver acording to configuration, I dont believe they are needed and seem to be unable to have them generated to propose we go with their removal